src/main/resources/skillresponse/response.properties

```diff
@@ -116,13 +119,16 @@ askForCityAfterStop.personalized={0}, You haven''t set your region yet. In what
 duplicatedStops.personalized=We found {0} stops associated with the stop number. Did you mean the {1} stop?
 #{0} stopName
 duplicatedStopConfirm.personalized=Ok, what about the {0} stop?
+#{1} cityName, {2} stopNumber
+copyProfile.personalized=Hi {0}, We found an existing profile living in {1} and the stop number is {2}, do you want to use that?
```

Comment:
Let's change this to give a better explanation that this is related to new support for personalization:

`...OneBusAway now supports personalized profiles, so let's set you up. Your current settings have you living in {1} and the stop number is {2}, do you want to use that?`

Reply:
In that case, if user answer no, user will hear  "Welcome to OneBusAway, let's set you up....." again, How about just `OneBusAway now supports personalized profiles, your current settings have you living in {0} and the stop number is {1}, do you want to use that?`

Reply:
Yes, that's close to what I meant - let's use:

`Hi {0}, OneBusAway now supports personalized profiles. Your current settings have you living in {1} and the stop number is {2}, do you want to use that?`